Lib/test/test_warnings/__init__.py

```diff
@@ -941,10 +941,10 @@ def func():
             {fname}:5: ResourceWarning: unclosed file <...>
               f = None
             Object allocated at (most recent call first):
```

Comment:
Hum, the message stil says "most recent call first". It seems like you have to patch Lib/warnings.py too.